app/assets/javascripts/cited_documents.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable camelcase */
+/* global CitedDocuments */
+/* global Blacklight */
+
+(function (global) {
+  var CitedDocuments;
+
+  CitedDocuments = {
+    init: function (el) {
+      var $el = $(el);
+      var data = $el.data();
+      var solrQueryString = data.documentids.reduce(function(query, nextId, index, array){
+        return (index < array.length - 1) ? query + nextId + ' OR ' : query + nextId;
+      }, '');
```

Comment:
is there a `.join` method for JavaScript arrays?

Reply:
Ha, yes. Not sure why reduce occurred to me first.